src/config/entities/__tests__/configuration.ts

```diff
@@ -171,7 +164,7 @@ export default (): ReturnType<typeof configuration> => ({
   express: { jsonLimit: '1mb' },
   features: {
     richFragments: true,
-    email: true,
+    email: false,
```

Comment:
This was set to `false` because otherwise, every test would have to include the now optional Alerts configuration (resulting in more changes).

Since this feature is opt-in, each test should decide if it should include the feature or not.